metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -19,7 +18,7 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class BigqueryTableType(Enum):
+class BigqueryTableType:
```

Comment:
we could still use StrEnum?

Reply:
Saw this, don't think it'll work here unless I change the queries -- that's what's actually failing, where I directly embed `BigQueryTableType.MATERIALIZED_VIEW` etc. in query text. If I keep as an Enum, then I have to change to `BigQueryTableType.MATERIALIZED_VIEW.value`, but I didn't see any value in keeping it as an enum when it's basically just a string const holder.

```
In [2]: from enum import Enum

In [3]: class Mine(str, Enum):
   ...:     A = "ONE"
   ...:     B = "TWO"
   ...: 

In [4]: Mine.A
Out[4]: <Mine.A: 'ONE'>

In [5]: Mine.A == "ONE"
Out[5]: True

In [6]: str(Mine.A)
Out[6]: 'Mine.A'

In [7]: str(Mine.A.value)
Out[7]: 'ONE'
```

Reply:
ugh didn't realize that it overrode the `__str__`

yep let's leave it like this then